config/tests/chime_verify_upchan.yaml

```diff
@@ -0,0 +1,330 @@
+    value: 153
+    kotekan_stage: testDataGen
+    out_buf: upchan_in
+
+write:
+  kotekan_stage: rawFileWrite
+  #in_buf: upchan_in
+  in_buf: network_buffer_0
+  file_name: 'upchan_in'
+  file_ext: 'dump'
+  base_dir: '.'
+
+cpu_simulate:
+  cpu_data_sim_0:
+    gain_dir: "./"
+    ew_spacing: [0.0, 0.1, 0.2, 0.3]
+    northmost_beam: 90.0
+    kotekan_stage: gpuBeamformSimulate
+    network_in_buf: network_buffer_0
+    beam_out_buf: cpu_output_buffer_0
+    hfb_out_buf: cpu_hfb_output_buffer_0
+
+compress_lost_samples:
+  kotekan_stage: compressLostSamples
+  compression_factor: factor_upchan * downsample_time
+  zero_all_in_group: true
+  in_buf: lost_samples_buffer
+  out_buf: compressed_lost_samples_buffer
+
+copy:
+  kotekan_stage: bufferCopy
+  in_buf: compressed_lost_samples_buffer
+  out_bufs:
+    - compressed_lost_samples_buffer_cpy
+
+gpu:
+  kernel_path: "../lib/hsa/kernels/"
+  commands: &command_list
+    - name: hsaInputData
+    - name: hsaHostToDeviceCopy
+      in_buf: hfb_compressed_lost_samples_buf
+      gpu_memory_name: hfb_compressed_lost_samples
+      #    - name: hsaHostToDeviceCopy
+      #      in_buf: upchan_buf
+      #      gpu_memory_name: upchan
+    - name: hsaOutputDataZero
+    - name: hsaAsyncCopyGain
+    - name: hsaBarrier
+    - name: hsaBeamformReorder
+    - name: hsaBeamformKernel
+    - name: hsaBeamformTranspose
+#   - name: hsaKVFFT
+    - name: hsaBeamformUpchanHFB
+    - name: hsaBeamformHFBSum
+    - name: hsaBeamformOutputData
+    - name: hsaBeamformHFBOutputData
+  enable_delay: true
+  cpu_affinity: [2, 3, 8, 9]
+  delay_max_fraction: 2.0
+  block_size: 32
+  buffer_depth: 1
+  n_intg: 24576
+  frame_arrival_period: samples_per_data_set / 390625
+  gpu_0:
+    kotekan_stage: hsaProcess
+    gpu_id: 0
+    ew_spacing: [0.0, 0.1, 0.2, 0.3]
+    northmost_beam: 90.0
+    commands: *command_list
+    in_buffers:
+      network_buf: network_buffer_0
+      gain_frb_buf: gain_frb_buffer
+      hfb_compressed_lost_samples_buf: compressed_lost_samples_buffer_cpy
+      #      upchan_buf: upchan_in
+    out_buffers:
+      beamform_output_buf: gpu_beamform_output_buffer_0
+      beamform_hfb_output_buf: gpu_beamform_hfb_output_buffer_0
+
+hyper_fine_beam:
+  num_frames_to_integrate: 4
+  good_samples_threshold: 0.95
+  integrate_hfb_data:
+    kotekan_stage: HFBAccumulate
+    log_level: debug
+    hfb_input_buf: gpu_beamform_hfb_output_buffer_0
+    compressed_lost_samples_buf: compressed_lost_samples_buffer
+    hfb_output_buf: beamform_hfb_output_buffer_0
+  
+hfb:
+    file_length: 1
+    file_type: hfbraw
+    root_path: .
+    kotekan_stage: HFBWriter
+    in_buf: beamform_hfb_output_buffer_0
+    instrument_name: chimeHFB
+
+read_gain:
+    kotekan_stage: ReadGain
+    updatable_config:
+      gain_frb: /updatable_config/frb_gain
+      gain_tracking: /updatable_config/tracking_gain
+    in_buf: network_buffer_0
+    gain_frb_buf: gain_frb_buffer
+    gain_tracking_buf: gain_tracking_buffer
```

Comment:
Indentation is inconsistent.   We seem to mostly use 2 spaces in yaml, so let's stick to that.